init-softlayer.html.md.erb

```diff
@@ -63,83 +68,135 @@ jobs:
   - {name: blobstore, release: bosh}
   - {name: director, release: bosh}
   - {name: health_monitor, release: bosh}
+  - {name: powerdns, release: bosh}
   - {name: softlayer_cpi, release: bosh-softlayer-cpi}
 
   resource_pool: vms
   persistent_disk_pool: disks
 
   networks:
-  - {name: private, static_ips: [10.0.0.6]}
+  - name: default
 
   properties:
     nats:
-      address: 127.0.0.1
       user: nats
       password: nats-password
-
-    postgres: &db
-      listen_address: 127.0.0.1
-      host: 127.0.0.1
+      auth_timeout: 3
+      address: 127.0.0.1
+      listen_address: 0.0.0.0
+      port: 4222
+      no_epoll: false
+      no_kqueue: true
+      http:
+        port: 9222
+
+    postgres: &20585760
       user: postgres
       password: postgres-password
+      host: 127.0.0.1
       database: bosh
       adapter: postgres
 
     blobstore:
-      address: 10.0.0.6
+      address: 127.0.0.1
+      director:
+        user: director
+        password: director-password
+      agent:
+        user: agent
+        password: agent-password
       port: 25250
       provider: dav
-      director: {user: director, password: director-password}
-      agent: {user: agent, password: agent-password}
 
     director:
-      address: 127.0.0.1
-      name: my-bosh
-      db: *db
       cpi_job: softlayer_cpi
-      max_threads: 4
-      user_management:
-        provider: local
-        local:
-          users:
-          - {name: admin, password: admin}
-          - {name: hm, password: hm-password}
+      address: 127.0.0.1
+      name: bosh
+      db:
+        adapter: postgres
+        database: bosh
+        host: 127.0.0.1
+        password: postges-password
+        user: postgres
+      enable_virtual_delete_vms: true
 
     hm:
-      director_account: {user: hm, password: hm-password}
-      resurrector_enabled: true
+      http:
+        user: hm
+        password: hm-password
+        port: 25923
+      director_account:
+        user: admin
+        password: admin-password
+      intervals:
+        log_stats: 300
+        agent_timeout: 180
+        rogue_agent_alert: 180
+
+    dns:
+      address: 127.0.0.1
+      domain_name: microbosh
+      db: *20585760
+      webserver:
+        port: 8081
+        address: 0.0.0.0
 
-    #TODO: add all required and optional props here and note optional
-    softlayer: &softlayer # <--- Replace values below
-      user_id: SOFTLAYER-USER-ID
-      api_key: SOFTLAYER-API-KEY
-      datacenter: SOFTLAYER-DATACENTER-NAME
+    softlayer: &softlayer
+      username: SOFTLAYER-USER-ID # <--- Replace with your softlayer user id
+      apiKey: SOFTLAYER-API-KEY # <--- Replace with your softlayer api key
 
-    agent: {mbus: "nats://nats:nats-password@10.0.0.6:4222"}
+    agent: {mbus: "nats://nats:nats-password@10.1.2.3:4222"}
```

Comment:
what's 10.1.2.3? please run bosh-init deploy with manifest from the docs to ensure that it works.